resources/views/vendors/bills/purchase_order/mail.blade.php

```diff
@@ -0,0 +1,16 @@
+
+@component('mail::message')
+Good day,
+
+This is your purchase order.
+
+@component('mail::table')
+| Due Date | Sub Total | Tax | Grand Total |
+| -------- |:----------:|----:| -----------:|
+| {{$purchase_order['due_date']}} | {{$purchase_order['sub_total']}} | {{$purchase_order['tax']}} | {{$purchase_order['grand_total']}} |
```

Comment:
Make it something like this:
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |
```
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |
```